Interop/Indexer/prebuilt/nativeInteropStubs/cpp/signalChaining.cpp

```diff
@@ -0,0 +1,58 @@
+#if defined(__linux__) || defined(__APPLE__)
+#include <dlfcn.h>
+#include <stdio.h>
+#include <signal.h>
+#if defined(__linux__)
+#include <link.h>
+#endif
+
+typedef void (*toggleCrashRecovery_t)(int);
+typedef int (*sigaction_t)(int, const struct sigaction *, struct sigaction *);
+
+static sigaction_t realSigaction = 0;
+
+static int mySigaction(int sig, const struct sigaction *act, struct sigaction * oact) {
+  if (sig == SIGILL || sig == SIGFPE|| sig ==  SIGSEGV || sig == SIGBUS || sig == SIGUSR1|| sig == SIGUSR2) return 0;
+  return realSigaction(sig, act, oact);
```

Comment:
What's the purpose of allowing libclang to handle other signals?
Seems unreliable because JVM can eventually rely on other signals too.

Reply:
Mostly for termination handlers. Remember, race comes from the high frequency signal used for safepointing, generally clang chains OKish. Which signals you have in mind.

Reply:
> Which signals you have in mind.

Any other signal JVM may find useful to handle someday.

Reply:
This part doesn't change for decade and the half, and if it will happen - we'll notice :).